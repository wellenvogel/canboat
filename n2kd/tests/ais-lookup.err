INFO test-n2kd [n2kd] Timestamp fixed
INFO test-n2kd [n2kd] New device src=23 seen
INFO test-n2kd [n2kd] New device src=23 seen
INFO test-n2kd [n2kd] New device src=23 seen
INFO test-n2kd [n2kd] New device src=23 seen
INFO test-n2kd [n2kd] New device src=23 seen
INFO test-n2kd [n2kd] New device src=23 seen
INFO test-n2kd [n2kd] New device src=23 seen
INFO test-n2kd [n2kd] New device src=23 seen
INFO test-n2kd [n2kd] New device src=23 seen
INFO test-n2kd [n2kd] New device src=23 seen
INFO test-n2kd [n2kd] New device src=23 seen
INFO test-n2kd [n2kd] N2KD stopping
