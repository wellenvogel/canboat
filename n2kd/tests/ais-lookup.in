2022-09-10T12:07:00.220Z,6,129794,23,255,76,05,80,26,aa,0e,f8,8e,8c,00,50,43,48,51,20,00,ff,55,4b,31,34,37,20,4d,4f,52,47,45,4e,53,54,45,52,00,ff,ff,ff,1e,a4,01,50,00,28,00,40,01,31,4a,00,00,00,00,e0,01,46,49,53,48,49,4e,47,20,47,52,4f,55,4e,44,53,20,20,20,00,ff,3d,e1,ff
2022-09-10T12:07:00.261Z,6,129794,23,255,76,05,10,fd,3b,0c,00,00,00,00,4f,50,48,45,20,20,20,5a,31,38,37,20,47,52,49,45,54,4a,45,20,20,20,20,20,20,20,20,1e,f0,00,46,00,32,00,8c,00,31,4a,00,00,00,00,5e,01,00,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,05,e1,ff
2022-09-10T12:07:04.236Z,6,129794,23,255,76,05,81,e6,96,0e,00,00,00,00,50,43,32,31,39,32,00,46,52,41,4e,53,20,20,48,4f,52,4a,55,53,00,ff,ff,ff,ff,ff,ff,45,40,01,46,00,0a,00,0e,01,31,4a,00,00,00,00,14,00,52,4f,4e,44,4a,45,20,5a,45,49,4c,45,4e,00,ff,ff,ff,ff,ff,ff,3d,e1,ff
2022-09-10T12:07:06.043Z,6,129794,23,255,76,05,75,9a,8b,0e,00,00,00,00,50,44,32,36,33,30,00,50,38,35,20,20,20,20,00,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,63,fa,00,32,00,1e,00,82,00,31,4a,00,00,00,00,00,00,00,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,3d,e0,ff
2022-09-10T12:07:10.619Z,6,129794,23,255,76,05,cd,49,8b,0e,da,a6,95,00,50,42,39,38,35,37,20,57,49,4c,4c,45,4d,20,42,41,52,45,4e,54,53,5a,20,20,20,20,20,3c,bc,02,aa,00,50,00,d2,00,31,4a,00,00,00,00,00,00,54,45,52,53,43,48,45,4c,4c,49,4e,47,20,20,20,20,20,20,20,20,06,e0,ff
2022-09-10T12:10:09.152Z,6,129794,23,255,76,05,68,af,95,0e,00,00,00,00,50,42,35,39,36,36,00,5a,57,41,52,54,45,20,56,41,4c,4b,40,40,40,40,40,40,40,40,41,b2,18,01,3c,00,32,00,e6,00,31,4a,00,00,00,00,0a,00,4d,41,4b,4b,55,4d,00,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,3d,e1,ff
2022-09-10T12:06:57.741Z,4,129038,23,255,28,01,81,e6,96,0e,eb,4d,3a,03,f7,80,b1,1f,e4,00,00,00,00,e4,10,09,ff,ff,ff,7f,c8,f8,ff
2022-09-10T12:06:58.568Z,4,129039,23,255,27,12,68,94,9c,0c,5a,ac,37,03,6f,df,9e,1f,e4,04,6b,05,00,06,00,0e,ff,ff,00,74,ff,ff
2022-09-10T12:07:03.527Z,6,129809,23,255,27,18,55,81,97,0e,57,49,54,54,45,20,52,41,41,46,00,ff,ff,ff,ff,ff,ff,ff,ff,ff,e1,ff
2022-09-10T12:07:01.969Z,6,129810,23,255,35,18,f6,b5,9c,0c,25,53,52,54,45,51,23,2d,44,44,37,35,35,38,00,82,00,28,00,0a,00,64,00,00,00,00,00,03,e0,ff
2022-09-10T12:10:33.092Z,4,129038,23,255,28,03,cd,49,8b,0e,45,37,39,03,d5,d9,b1,1f,81,bc,33,86,01,00,20,00,08,2f,cc,ff,c0,f8,ff
