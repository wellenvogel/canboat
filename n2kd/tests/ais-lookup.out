!AIVDM,2,1,1,B,53b`VP42<SgQ0<Q4001Dg7CN0lu8LDq=@E80000N40:44t0Ht<1RDj2CQp1l,0*17
!AIVDM,2,2,1,B,SmCQ4h00000,2*56
!AIVDM,2,1,2,B,533gu4400000u0PD001c7SN0M8TE@`D00000000N1h:2540Ht8h000000000,0*68
!AIVDM,2,2,2,B,00000000000,2*25
!AIVDM,2,1,3,B,53aKVPD000010?;7W80I84q>20Pu8aE<000000153H561t0Ht0TSkQ2QH6QB,0*2D
!AIVDM,2,2,3,B,C1CP0000000,2*45
!AIVDM,2,1,4,A,53`fJMD000010C;K?013SD00000000000000001S1`<23t0Ht00000000000,0*3C
!AIVDM,2,2,4,A,00000000000,2*20
!AIVDM,2,1,5,A,53`e9kH2Eaea0;WSGL1LThhDn0858DqA=`00000t2`i9840Ht051DThj1C32,0*23
!AIVDM,2,2,5,A,CQh00000000,2*6B
!AIVDM,2,1,6,B,53aFgJ4000010;GWKH1aL59@F1H4hd000000006j2p515t0Ht0C@BjmC@000,0*44
!AIVDM,2,2,6,B,00000000000,2*21
!AIVDM,1,1,,B,13aKVPHP000HjPBNK?Jh0?wj0000,0*19
!AIVDM,1,1,,B,B39jDJ000@6;G?WUdCuR;wtUj006,0*3F
!AIVDM,1,1,,B,H3aN1EALUA@F1844H00000000000,0*6B
!AIVDM,1,1,,A,H39jmuTUCBD5ASe44ommp01@331000000,0*7F
!AIVDM,1,1,,A,33`e9k@uA<PHhJRNKDh2uj;00000,0*5E
